spec/javascripts/actions/story_spec.js

```diff
@@ -512,4 +513,39 @@ describe('Story Actions', () => {
       });
     });
   });
+
+  describe('toggleEpic', () => {
+    const projectId = 1;
+    const stories = Array(3).fill(storyFactory());
+    const fakeGetState = jest.fn();
```

Comment:
It's good to avoid sharing mutable state across multiple tests, what do you think about moving these to inside the `beforeEach` block as we did for `fakeDispatch`?